src/common/Utilities/Util.cpp

```diff
@@ -392,22 +392,22 @@ std::wstring GetMainPartOfName(std::wstring const& wname, uint32 declension)
     }
 
     // Important: end length must be <= MAX_INTERNAL_PLAYER_NAME-MAX_PLAYER_NAME (3 currently)
-    static std::wstring const a_End    = { wchar_t(0x0430), wchar_t(0x0000) };
```

Comment:
Endings are initialized with a wchar_t(0x0000) at the end. This is not necessary and might cause issues with string comparisons. In C++, strings are null-terminated automatically, so we don't need to include the null character explicitly.